src/planner/macos.rs

```diff
@@ -110,9 +118,10 @@ impl Planner for Macos {
             },
         };
 
-        let encrypt = match self.encrypt {
-            Some(choice) => choice,
-            None => {
+        let encrypt = match (self.nix_enterprise, self.encrypt) {
```

Comment:
Encrypt doesn't seem used in the `self.nix_enterprise == true` case, so why do we need to match on it here?

Reply:
I did it this way to be clearer in the receipt, so encrypt: true in the .json. I left a comment about why this code is a bit goofy ...